src/workerd/api/web-socket.c++

```diff
@@ -574,17 +578,19 @@ void WebSocket::close(
   if (reason != kj::none) {
     // The default code of 1005 cannot have a reason, per the standard, so if a reason is specified
     // then there must be a code, too.
-    JSG_REQUIRE(code != nullptr, TypeError,
+    JSG_REQUIRE(code != kj::none, TypeError,
         "If you specify a WebSocket close reason, you must also specify a code.");
   }
 
+  auto n = autoResponseStatus.pendingAutoResponseQueue.size() - autoResponseStatus.queuedAutoResponses;
```

Comment:
non-blocking nit: a doc comment explaining this a bit would be helpful here but not critical.